.github/workflows/main.yaml

```diff
@@ -30,6 +30,6 @@ jobs:
       - run: tox -e lint
       - run: tox -e py
       - run: shopt -s globstar && pyupgrade --py3-only **/*.py # --py36-plus
-      - run: safety check -i 42559 # pip <= 20.1.1, we upgrade it just above
+      - run: safety check -i 42559 -i 62044 # pip <= 20.1.1, we upgrade it just above
```

Comment:
I think I have just discovered myself what this is 

My commit for version 5.12.3 fails because of what I presume is the same problem https://github.com/burnash/gspread/actions/runs/7216614506/job/19663108034

What have you learnt?

Reply:
Yes this is a tool to check for known security issues. 

I am telling it to ignore `pip` related issues.

Reply:
> I think I have just discovered myself what this is
> 
> My commit for version 5.12.3 fails because of what I presume is the same problem https://github.com/burnash/gspread/actions/runs/7216614506/job/19663108034
> 
> What have you learnt?

Yes you commit and the one from dependabot too.
This is because now the `pip` tool has a known security issue. I chosed to ignore it as `pip` is only a tool for the CI if has issues we don't care so much we don't use it in the library.

Reply:
having a safety check, that we ignore the output of whenever there is any, seems a strange idea to me. but fair enough

Reply:
I know it's not a good practice, when the safety check is about your software/library/deployement stack etc. here... we don't handle pip version, we can force users to use pip-22 or pip-23 and it must not impact us.

So We can safely ignore it here to me. 

We can try later to upgrade pip to newer version etc in a better way so may be the check goes away ?
it is quiet common with safety check that until a fix is released on the given package, you face the error message.